### PR TITLE
Merge branch search dashboard ng2017

### PR DESCRIPTION
**Pull request by wlach**
This is just a rollup of everything we've committed so far, just-rebased against master.

### CODE REVIEW

File: mozetl/search/aggregates.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+def search_aggregates(main_summary):
+    return agg_search_data(
+        main_summary,
+        [
+            'country',
+            'engine',
+            'source',
+            'submission_date',
+            'country',
```
Comment (harterrt) [inline mozetl/search/aggregates.py]: Oops, country is listed twice in this array and causes an error when running on live data.

Comment (harterrt) [inline mozetl/search/aggregates.py]: Country is listed twice, but should be listed once.

**Comment by wlach** [comment]
@harterrt I'll fix things up, give me a few minutes :)

File: mozetl/search/aggregates.py (hunk 1 above)
Comment (wlach) [reply mozetl/search/aggregates.py]: Can we order these alphabetically? That would make errors like this easier to spot.

**Comment by wlach** [comment]
@harterrt take a look now, still a pending question on whether it's ok to sort the columns alphabetically (I would be surprised if it wasn't?)

File: mozetl/search/aggregates.py (hunk 1 above)
Comment (harterrt) [reply mozetl/search/aggregates.py]: Great idea.

I believe that will affect the default column order of the resulting dataset, but that is a non-issue.

File: mozetl/search/aggregates.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                       save_mode=DEFAULT_SAVE_MODE):
+    start = datetime.datetime.now()
+    spark = (
+        SparkSession
+        .builder
+        .appName('search_dashboard_etl')
+        .getOrCreate()
+    )
+
+    version = 2
```
Comment (harterrt) [inline mozetl/search/aggregates.py]: Looks like this will be version 3 of the dashboard. Can you bump this as well?

**Comment by harterrt** [comment]
Backfill complete. Waiting on parquet2hive to refresh to test the dataset in re:dash.

**Comment by harterrt** [comment]
And we're live - STMO is reporting the new schema: https://sql.telemetry.mozilla.org/queries/50570/source

**Pull request merged**